Bottom: dc84f776625d1c5b119ae19d2baa4ebfa0b05c09
Top:    84b34687065d52f1b5cd9843fd3ef753d5393ed4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-28 15:49:56 -0500

Refresh of add-docker-option

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 74c6785b..9b9c6261 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -61,7 +61,8 @@
     - openssl_container
   - name: Build and start Nexus Artifact Repository container
     role: nexus3
-    tags: [nexus, container]
+    tags:
+    - nexus_container
   - name: Build and start the SVN Docker container
     role: docker-svn
     svn_repos: 'ddio NCD_Release'
